gemrb/core/Streams/DataStream.h

```diff
@@ -153,7 +153,7 @@ class GEM_EXPORT DataStream {
 	
 	template <typename STR>
 	strret_t WriteString(const STR& src, size_t len) {
-		return Write(src.begin(), len);
+		return Write(src.c_str(), len);
```

Comment:
Here, at some point the compiler started complaining of how `begin()` won't go into `void*`.